test/test_tableMapper.js

```diff
@@ -0,0 +1,55 @@
+var assert = require('assert');
+var mapper = require('../mappers/tableMapper');
+var mongoose = require('mongoose');
+const APP_DB = 'mongodb://127.0.0.1/eventplanner_db';
+
+describe('tableMapper testing suite', function() {
+	//After all tests reconnect to APP_DB
+	before(function(){
+		mongoose.connect(APP_DB);
+	});
+
+	it('should create a new table', function(done) {
+		mapper.createTable(12,[],5,['a','b','c','d','e'],1,function(err,res){
+			assert.equal(err,null);
+			assert.equal(res.tableNumber,12);
+			done();
+		});
+	});
+
+	it('should find tables by eventId', function(done) {
+		mapper.findTablesByEventId(1,function(err,result){
+			assert.notEqual(result[0].tableNumber,0);
```

Comment:
What exactly is this unit test testing? That at least one table has been created?

Reply:
Bit confused! Since result[0] will always be table number =1 if it exists. But not necessarily finding all tables

Reply:
Should have been result.length != 0